### PR TITLE
Upgrade core to 1.152.1

### PR DESCRIPTION
**Pull request by nicodh**
See https://github.com/deltachat/deltachat-core-rust/issues/6338
and https://github.com/deltachat/deltachat-core-rust/pull/6346

### CODE REVIEW

**Review by hpk42 — APPROVED** [review]
lgtm

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,8 +5,10 @@
 ## Added
 
 ## Changed
+- Update `@deltachat/stdio-rpc-server` and `deltachat/jsonrpc-client` to `1.152.1`
 
 ## Fixed
+- downgrade rust in core to avoid wrong Windows malware detection https://github.com/deltachat/deltachat-core-rust/issues/6338
```
Comment (WofWca) [inline CHANGELOG.md:11]: Looking at previous entries, we seem to usually bunch together changelog entries for Core upgrades

**Review by WofWca — APPROVED** [review]
I uploaded the rpc.exe file to VirusTotal from this build just in case, and it came out clean.

File: CHANGELOG.md (hunk 1 above)
Comment (nicodh) [reply CHANGELOG.md:11]: Well between 1.152.0 and 1.152.1 there is nothing else than what is in this line

**Pull request merged**